Contributing.md

```diff
@@ -0,0 +1,88 @@
+
+```{bash}
+bundle install
+```
+
+install the Gulp dependencies
+
+```{bash}
+npm i
+```
+
+`Gulpfile` has two main tasks: `build` and `watch`
+
+`build` does just that - builds the `_site` directory.
+
+`watch` watches both `assets/css/_inc` directory to check for CSS changes as well as Jekyll file changes and rebuilds the appopriate files.
+
+**A note about the CSS** All CSS is in the `_inc` directory as plain CSS. Current browser support for variables is strong, but we use PostCSS to fill in browser compatibility. PostCSS also compiles `assets/css/_inc/main.css` into `assets/css/main.css`. You should never need to edit `assets/css/main.css` file directly.
+
+Reminder that this is a [Jekyll theme](https://jekyllrb.com/docs/themes/), not a stand alone site. We optimized the build tools to both allow customization by the individual response sites by overriding `main.css` locally as well as is.
+
+## Where to start?
+
+The [issues](https://github.com/hurricane-response/response-theme/issues). We ask that you create an issue before starting work on something, even if a new feature. We also ask that you adhere as best as possible to the issue template to help us better triage. Certainly small,minor typos do not require a novel, but the more information the better.
+
+We have found that robust discussion before starting a new feature or large refactor has the greatest chance of success. Discussions can occur in the issue created or in Slack.
+
+
+**Note** sometimes during an active response focus is on immediate needs and those of our partners. We respectfully ask that you use the issues and `@` one of the contributors.
+
+### Whoa! I just want to fix a typo.
+
+Yes! We understand that frustration. GitHub does a great job of providing an in-browser experience for this scenario. If you don't have a GitHub account, first [create an account](https://github.com/join) and then navigate to the file with the minor change. In the upper right corner, next to the trash can symbol is a pencil. That icon will allow you to start the editing process. GitHub automagically will create a fork with your change and walk you through submitting it as a pull request.
+
+## Commit Messages
+
+The message should be a short summary (max. 50 chars) written in the imperative, followed by an optional, more detailed explanatory text separated from the summary by an empty line.
+
+Commit messages should follow best practices, including explaining the context of the problem and solution, including in caveats or follow up changes required. They should tell the story of the change and provide readers understanding of what led to it.
+
+If you're lost about what this even means, please see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) for a start.
+
+
+## Pull Requests
+
+All Pull Requests should have an accompanying issue. We realize during an active response this policy is fluid and we do require peer review for all PRs.
```

Comment:
I'd get rid of the "We realize" piece, and just state this as a "Note" as was done above in the **Where to start?** section. Perhaps something like:

```
All Pull Requests should have an accompanying issue and will require a review prior to merging.

**Note: During an active response, this policy can be fluid, as things change quickly and fixes are often needed immediately. During those times, be sure to get an explicit OK or instruction from a maintainer before creating a PR not tied to an issue.

Only maintainers can merge PRs, and only they can decide whether to waive or defer a review on a PR. As a rule, this doesn't happen often, and only does during an active response or major service outage.**
```

Reply:
I dig the phrasing here (no opinion on where it should be), and let me suggest to even further emphasize the overcommuincation aspect. Something like "During an active response, we strongly encourage you to check in with a maintainer via Slack before beginning any work (especially large changes) to prevent code conflicts and ensure you're working on the most important thing."

Reply:
There's another section 

> We have found that robust discussion before starting a new feature or large refactor has the greatest chance of success. Discussions can occur in the issue created or in Slack.

> **Note** sometimes during an active response focus is on immediate needs and those of our partners. We respectfully ask that you use the issues and `@` one of the contributors.

that's in "where to start?" section.